.github/workflows/_python-tests.yml

```diff
@@ -104,7 +104,7 @@ jobs:
         api_key: ${{ secrets.DD_API_KEY }}
         site: ${{ vars.DD_SITE }}
     - name: Rust Integration Test
-      run: bin/rust-integration-test.sh ${{ matrix.test-globs }}
+      run: LIBRARY_PATH=$LD_LIBRARY_PATH bin/rust-integration-test.sh ${{ matrix.test-globs }}
```

Comment:
`setup-python` doesn't properly configure env vars to point to the right version. We have our own `buildpulse/setup-python` that will take care of this.